apps/web/cypress/support/commands.ts

```diff
@@ -71,7 +71,11 @@ Cypress.Commands.add('clickNodeButton', (selector: string) => {
 
 Cypress.Commands.add(
   'initializeSession',
-  (settings: { disableLocalStorage?: boolean; noTemplates?: boolean } = { disableLocalStorage: false }) => {
+  (
+    settings: { disableLocalStorage?: boolean; noTemplates?: boolean; showOnBoardingTour?: boolean } = {
+      disableLocalStorage: false,
+    }
+  ) => {
```

Comment:
Fixed the issue with flaky e2e tests that were failing in the case when the "start from the scratch tour" was shown which was covering the workflow editor drop-box, so the drop action couldn't perform.